common/src/main/java/com/wynntils/mc/event/ChatScreenSendEvent.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.event;
+
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.eventbus.api.Event;
+
+@Cancelable
+public class ChatScreenSendEvent extends Event {
```

Comment:
We refer to dialogue windows as "chat screen". Why not just call it ChatSendEvent?

Reply:
ChatSendEvent and ChatSentEvent? That seems kind of suspicious and a very easy way to cause problems in the future...

Reply:
What would be the difference between the two events?